zwave_js_server/dump.py

```diff
@@ -16,8 +18,24 @@ async def dump_msgs(
     msgs.append(version)
 
     await client.send_json({"command": "start_listening"})
-    state = await client.receive_json()
-    msgs.append(state)
+    msg = await client.receive_json()
+    msgs.append(msg)
+    state = msg["result"]["state"]
+
+    # If it's None, old version of the server, ignore it.
+    if wait_nodes_ready and state["driver"].get("allNodesReady") is False:
```

Comment:
Old version of server raises an error since the key doesn't exist.

```
  File "/src/zwave_js_server/dump.py", line 29, in dump_msgs
    if wait_nodes_ready and state["driver"].get("allNodesReady") is False:
KeyError: 'driver'
```